src/App.tsx

```diff
@@ -52,6 +52,15 @@ function App() {
           as="textarea"
           value={textarea}
           placeholder={defaultText}
+          onKeyDown={(e: any)=> {
+            if (e.keyCode === 9) {
```

Comment:
This will just add a tab to the end of the text, rather than adding a tab where the cursor is. You would need to figure out where the cursor is positioned in the text and add it there

Reply:
yep i am fixng that one 😄